README.md

```diff
@@ -17,53 +17,63 @@
 
 Core:
 
- - GNU make (try gmake if make fails)
- - GCC 4 or later
+* GNU make
+* autoconf and automake
+* pkg-config
+* GCC 4 or clang
 
 ## Optional Requirements
 
-SSL support:
-
- - openssl 0.9.7d or later (try installing openssl-dev, openssl-devel or
-   libssl-dev)
+SSL/TLS support:
+* openssl 0.9.7d or later
+    * try installing openssl*dev, openssl*devel or libssl*dev
+    * Mac OS X: OpenSSL from Homebrew is prefered over system
+        * use `--USE_SYSTEM_OPENSSL` to force `configure` to use the (deprecated) Mac OS X version
 
 modperl:
-
- - This needs perl and its bundled libperl
+* perl and its bundled libperl
 
 modpython:
-
- - This needs perl(!) and python's bundled libpython
+* python and its bundled libpython
+* modperl is required
+* Mac OS X: Python from Homebrew is preferred over system version
 
 cyrusauth:
+* This module needs cyrus-sasl2
 
- - This module needs cyrus-sasl2
+Character Encodings:
+* To get proper character encoding and charsets install ICU (`libicu4-dev`)
 
 ## Installing ZNC
 
-If you are building from git, you will need to run `./autogen.sh` first to produce the `configure` script.
-Note that this requires `automake` and `gettext` to be installed.
+Installation from source code is performed via `automake` toolchain:
 
-Installation is done with the `./configure ; make ; make install` commands.
+```shell
+./autogen.sh
+./configure
+make
+make install
+```
 
 You can use
-	./configure --help
+	`./configure --help`
 if you want to get a list of options, though the defaults should be suiting
 most needs. After you compiled it with make (or gmake if make doesn't work) you
 can install it with
-	make install
+	`make install`
 though you don't need to as ZNC supports in-place execution.
 
 ## Setting up znc.conf
 
-For setting up a configuration file in `~/.znc` you can simply do
-	znc --makeconf
-or
-	./znc --makeconf
-for in-place execution.
+For setting up a configuration file in `~/.znc`:
+```shell
```

Comment:
The benefit of using markdown was that it looks beautiful in plain text too, not only after rendering :(
All the "build status" images and such code blocks breaks it...